automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterFixtureSpec.scala

```diff
@@ -89,13 +97,14 @@ abstract class ClusterFixtureSpec extends fixture.FreeSpec with BeforeAndAfterAl
 
   override def afterAll(): Unit = {
     logger.info("afterAll")
-    if (!debug) {
+    if (debug)
+      super.afterAll()
```

Comment:
Is this right? We'd want to call `afterAll` if NOT in debug mode, right?

Reply:
no...just fixed it :)